docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,153 @@ sidebar_position: 2
 ---
 
 import FeedbackComponent from "@site/src/pages/feedback.md";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-# Generating SessionSigs
+# Generating SessionSigs: `getSessionSigs`
```

Comment:
```suggestion
# Generating SessionSigs Using an Auth Sig
```

Reply:
Did this, but `AuthSig` or `Auth Sig`?

Reply:
Will make this Auth Sig, I think people will understand